### PR TITLE
Transport cleanup

### PR DESCRIPTION
**Pull request by istankovic**
This PR does the following:
- move all WebRTC related things from `MatrixTransport` to `WebRTCManager`, where they belong
- make public methods private as appropriate
- remove unneeded code

This results in improved separation between `MatrixTransport` and `WebRTCManager`.

### CODE REVIEW

File: raiden/network/transport/matrix/rtc/web_rtc.py [hunk 1]
```diff
@@ -362,70 +369,202 @@ async def reset(self) -> None:
...
+        try:
+            return self._initialize_web_rtc(address)
+        finally:
+            self._web_rtc_channel_inits.remove(address)
+
+    def _initialize_web_rtc(self, partner_address: Address) -> None:
+        rtc_partner = self.get_rtc_partner(partner_address)
+        # XXX-UAM: extract partner caps from metadata here
+        # XXX allow for Web-RTC again
+        is_to_device = False
```
Comment (fredo) [inline raiden/network/transport/matrix/rtc/web_rtc.py:469]: to_device is a requirement now. I think we do not check it anymore.

File: raiden/network/transport/matrix/transport.py [hunk 2]
```diff
@@ -522,9 +480,10 @@ def start(  # type: ignore
             raise RuntimeError(f"{self!r} already started")
         self.log.debug("Matrix starting")
         self._stop_event.clear()
-        self._starting = True
         self._raiden_service = raiden_service
-        self._web_rtc_manager.node_address = self._raiden_service.address
+        self._web_rtc_manager = WebRTCManager(
```
Comment (fredo) [inline raiden/network/transport/matrix/transport.py:484]: Dont you get a lint here for not initialized in __init__ ?

**Review by fredo — APPROVED** [review]
Overall looks good to me. thanks for the clean up. I left some comments though. 

And one thing I noticed which might be important to know but it's hard to see. We need to be really careful whenever calling callbacks which are NOT asyncio that those are wrapped by aiogevent. 

This is done implicitly by [create_task_callback](https://github.com/raiden-network/raiden/blob/114e239b7156f59028dcfe146a1a2a8be251cc03/raiden/network/transport/matrix/rtc/utils.py#L58) and explicitly by wrap_callback()

File: raiden/network/transport/matrix/transport.py (hunk 2 above)
Comment (istankovic) [reply raiden/network/transport/matrix/transport.py:484]: Apparently not, maybe due to some differences in tool version(s)?

**Pull request merged**